src/WPF/ArcGISRuntime.WPF.Samples/Samples/Data/RasterLayerGeoPackage/RasterLayerGeoPackage.xaml.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright 2017 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using ArcGISRuntime.Samples.Managers;
+using Esri.ArcGISRuntime.Data;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Rasters;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ArcGISRuntime.WPF.Samples.RasterLayerGeoPackage
+{
+    public partial class RasterLayerGeoPackage
+    {
+        public RasterLayerGeoPackage()
+        {
+            InitializeComponent();
+            
+            // Read data from the GeoPackage
+            Initialize();
+        }
+
+        private async void Initialize()
+        {
+            // Create a new map centered on Aurora Colorado
+            MyMapView.Map = new Map(BasemapType.LightGrayCanvas, 39.5517, -104.8589, 12);
+
+            // Get the full path
+            string geoPackagePath = await GetGeoPackagePath();
+
+            // Open the GeoPackage
+            GeoPackage myGeoPackage = await GeoPackage.OpenAsync(geoPackagePath);
+
+            // Read the raster images and get the first one
+            Raster geoPackageRaster = myGeoPackage.GeoPackageRasters.FirstOrDefault();
```

Comment:
It is best not to name a variable the same a an ArcGISRuntime Type. Ex: use myGeoPackageRaster not geoPackageRaster.